tests/opportunity.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { test, expect } from '@playwright/test';
+import { api } from './utils';
+import * as config from '../init/config';
+
+test.describe('/api/opportunities', () => {
```

Comment:
We should also test the get one endpoint (`/api/opportunities/:id`).

Reply:
hey @nikita-pardeshi-github I think as you created `"/api/opportunities/2"` you can mark this as resolved.